usr/src/cmd/zdb/zdb.c

```diff
@@ -1558,8 +1558,32 @@ dump_deadlist(dsl_deadlist_t *dl)
 static avl_tree_t idx_tree;
 static avl_tree_t domain_tree;
 static boolean_t fuid_table_loaded;
-static boolean_t sa_loaded;
-sa_attr_type_t *sa_attr_table;
+sa_attr_type_t *sa_attr_table = NULL;
+
+static int
+check_sa_attr_table(objset_t *os)
```

Comment:
would it make sense to name this "load_sa_attr_table"?  Since it is changing the global state to load it if it isn't already loaded?